core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.html

```diff
@@ -202,11 +200,10 @@ <h4 class="oppia-mobile-translation-modal-author-header">
 </div>
 
 <style>
-  .oppia-mobile-translation-modal-author-header {
-    display: none;
+  .oppia-border-class {
```

Comment:
Naming suggestion:

- Drop `class` from the class name.
- Add something to signify this is for reviewable messages

e.g. `oppia-reviewable-message`